File: VB/S171806_CustomArgs/CustomReportPreviewModel.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.Xpf.Printing
Imports DevExpress.XtraPrinting

Namespace S171806_CustomArgs
	Public Class CustomReportPreviewModel
		Inherits ReportPreviewModel
		Public Sub SaveDocumentOnServer()
			MyBase.Export(ExportFormat.Pdf, New ExportState With {.FileName = "TestDocument"})
		End Sub
	End Class
End Namespace

File: CS/S171806_CustomArgs/CustomReportPreviewModel.cs
using DevExpress.Xpf.Printing;
using DevExpress.XtraPrinting;

namespace S171806_CustomArgs {
    public class CustomReportPreviewModel : ReportPreviewModel {
        public void SaveDocumentOnServer() {
            base.Export(ExportFormat.Pdf, new ExportState { FileName = "TestDocument" });
        }
    }
}
